ppa/archive/views.py

```diff
@@ -32,6 +32,12 @@ class DigitizedWorkListView(ListView):
     # keyword query; assume no search terms unless set
     query = None
 
+    def get_template_names(self):
+        # allow querying just the results via ajax
```

Comment:
I would rephrase this comment to make it more specific, something like: when queried via ajax, return partial html for just the results section without the form